src/server/routes/api/v1/parking-lot/index.ts

```diff
@@ -0,0 +1,106 @@
+import httpResponse from '../../../../utils/http-response';
+import { UnexistentParkingLotID } from '../../../../error/parking-lot.service';
+import type {
+  FastifyError,
+  FastifyInstance,
+  FastifyRegisterOptions,
+  FastifyReply,
+  FastifyRequest,
+} from 'fastify';
+import validationSchema from './validation-schema';
+
+export default function (
+  fastify: FastifyInstance,
+  _: FastifyRegisterOptions<unknown>,
+  done: (err?: FastifyError) => void,
+): void {
+  fastify.post(
+    '/',
+    { schema: validationSchema.createParkingLotSchema },
+    async (
+      request: FastifyRequest<{
+        Body: {
+          name: string;
+        };
+      }>,
+      reply: FastifyReply,
+    ) => {
+      try {
+        const parkingLot = await fastify.services.parkingLot.create(
+          request.body,
+        );
+
+        return httpResponse.created(reply, parkingLot);
+      } catch (error) {
+        return httpResponse.internalServerError(reply, error);
+      }
+    },
+  );
+
+  fastify.post(
+    '/:id',
+    { schema: validationSchema.updateParkingLotSchema },
+    async (
+      request: FastifyRequest<{
+        Params: {
+          id: string;
+        };
+        Body: {
+          name: string;
+        };
+      }>,
+      reply: FastifyReply,
+    ) => {
+      try {
+        await fastify.services.parkingLot.findById(request.params.id);
+
+        const parkingLot = await fastify.services.parkingLot.update({
+          id: request.params.id,
+          ...request.body,
+        });
+
+        return httpResponse.created(reply, parkingLot);
```

Comment:
This is returning a `201` which is for creating resources instead of a `200` which is for successful responses.

```suggestion
        return httpResponse.ok(reply, parkingLot);
```